lib/linux/ui.nit

```diff
@@ -343,3 +343,12 @@ redef class TextInput
 		super
 	end
 end
+
+redef class Text
+	redef fun open_in_browser
+	do
+		var proc = new Process("xdg-open", self)
+		proc.wait
```

Comment:
Browsers are known to have a crappy behavior with `wait`. Often the process delegates to an existing user session then exists right away (`Created new window in existing browser session.`).
If there is no usable user session then the process becomes the main session process and will not exit util all tabs and windows are closed (so basically never if your app opens a Wikipedia article)